src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -105,14 +105,12 @@ impl LazyBatchColumnVec {
         self.columns[0].len()
     }
 
-    /// Debug asserts that all columns have equal length.
+    /// Asserts that all columns have equal length.
     #[inline]
-    pub fn debug_assert_columns_equal_length(&self) {
-        if cfg!(debug_assertions) {
-            let len = self.rows_len();
-            for column in &self.columns {
-                debug_assert_eq!(len, column.len());
-            }
+    pub fn assert_columns_equal_length(&self) {
```

Comment:
Why remote `debug`?

Reply:
Because it's cost is not notable. Worth to enable in release.

Reply:
Nop, this function is called only once per single batch, so it is no longer marked as debug_assert.